### PR TITLE
Edited text for clarity and flow.

### PR DESCRIPTION
**Pull request by MCamp859**
Signed-off-by: MCamp859 <mary.camp@ptiglobal.net>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,19 +6,21 @@
 
 - [About](#about)
 - [GPU device plugin](cmd/gpu_plugin/README.md)
-- [FPGA device plugin](cmd/fpga_plugin/README.md)
-- [FPGA admission controller webhook](cmd/fpga_admissionwebhook/README.md)
-- [FPGA prestart CRI-O hook](cmd/fpga_crihook/README.md)
+- FPGA plugin code
```
Comment (MCamp859) [inline README.md]: Is "code" the proper word here? Could be files or repositories or something else?

File: README.md [hunk 2]
```diff
@@ -6,19 +6,21 @@
...
 ## About
 
-This repository contains set of Intel Device Plugins that enable usage of Intel hardware in Kubernetes cluster.
+This repository contains a set of Kubernetes plugins that enable you to use
+Intel devices.
 
-These are prerequisites for building and running Intel Device Plugins:
+Prerequisites for building and running Intel device plugins include:
 
 - Intel hardware
```
Comment (MCamp859) [inline README.md]: Add a pointer or link to which Intel hardware is supported?

File: README.md (hunk 1 above)
Comment (rojkov) [reply README.md]: "code" sounds good to me.

File: README.md (hunk 2 above)
Comment (rojkov) [reply README.md]: So far we've tested the code with this device only https://www.intel.com/content/www/us/en/programmable/products/boards_and_kits/dev-kits/altera/kit-a10-gx-fpga.html

**Pull request merged**